ecommerce_integrations/shopify/order.py

```diff
@@ -34,11 +34,24 @@ def sync_sales_order(payload, request_id=None):
 	try:
 		shopify_customer = order.get("customer", {})
 		customer_id = shopify_customer.get("id")
+		customer_name = shopify_customer.get("first_name", "") + " " + shopify_customer.get("last_name", "")
 		if customer_id:
 			customer = ShopifyCustomer(customer_id=customer_id)
 			if not customer.is_synced():
 				customer.sync_customer(customer=shopify_customer)
 
+				if(shopify_customer.get("first_name") and shopify_customer.get("last_name") and shopify_customer.get("email")):
+					customer.create_contact(shopify_customer.get("first_name"),
+					shopify_customer.get("last_name"),
+					shopify_customer.get("email"),
+					shopify_customer.get("phone"),
+					"",
+					shopify_customer.get("accepts_marketing"))
+
+			else:
+				customer.update_additional_address(customer_name, shopify_customer.get("email"),"Billing", order.get("billing_address"))
+				customer.update_additional_address(customer_name, shopify_customer.get("email"),"Shipping", order.get("billing_address"))
```

Comment:
Both are `billing_address`. Seems like a bug.

Reply:
It is a bug...shit